packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -298,7 +298,7 @@ $accent-blue: #117ac9;
 		border-radius: 4px;
 		text-transform: unset;
 		// stylelint-disable-next-line scales/font-size
-		font-size: 12px; // typography-exception smaller size needed
+		font-size: $font-body-extra-small; // typography-exception smaller size needed
```

Comment:
Was wondering if the stylelint exception comment can now be removed. And also, what's the whole `typography-exception` here about (haven't digged into the context of this yet).

Reply:
`// stylelint-disable-next-line scales/font-size` is added because a non-variable font size was previously used.

And `// typography-exception` is added when a non-standard font size is used to justify why that non-standard font size is needed.

So these 2 comments can now be removed.